crates/ruff_linter/src/rules/ruff/rules/snapshots/ruff_linter__rules__ruff__rules__unreachable__tests__for.py.md.snap

```diff
@@ -238,4 +238,65 @@ flowchart TD
   block0 --> return
 ```
 
+## Function 8
+### Source
+```python
+def func():
+    for i in range(5):
+        pass
+    else:
+        return 1
+```
+
+### Control Flow Graph
+```mermaid
+flowchart TD
+  start(("Start"))
+  return(("End"))
+  block0["pass\n"]
+  block1["return 1\n"]
+  block2["for i in range(5):
+        pass
+    else:
+        return 1\n"]
+
+  start --> block2
+  block2 -- "range(5)" --> block0
+  block2 -- "else" --> block1
+  block1 --> return
+  block0 --> return
```

Comment:
`block0` here (`pass`)` should point to `block2` (`for`).

In the next example, you can see that `block1` (`pass`) gets pointed to `black3` (`for`). But the structures are identical. It has to do with the lack of a statement following the loop.